CITATION.cff

```diff
@@ -12,23 +12,27 @@ authors:
     family-names: Liu
     email: ziwen.liu@czbiohub.org
     affiliation: Chan Zuckerberg Biohub San Francisco
-    orcid: 'https://orcid.org/0000-0001-7482-1299'
+    orcid: "https://orcid.org/0000-0001-7482-1299"
   - given-names: Eduardo
     family-names: Hirata-Miyasaki
     affiliation: Chan Zuckerberg Biohub San Francisco
-    orcid: 'https://orcid.org/0000-0002-1016-2447'
+    orcid: "https://orcid.org/0000-0002-1016-2447"
   - given-names: Christian
     family-names: Foley
   - given-names: Soorya
     family-names: Pradeep
     affiliation: Chan Zuckerberg Biohub San Francisco
-    orcid: 'https://orcid.org/0000-0002-0926-1480'
+    orcid: "https://orcid.org/0000-0002-0926-1480"
+  - given-names: Alishba
+    family-names: Imran
+    affiliation: Chan Zuckerberg Biohub San Francisco
+    orcid: "https://orcid.org/0009-0003-7049-355X"
   - given-names: Shalin
     family-names: Mehta
     affiliation: Chan Zuckerberg Biohub San Francisco
-    orcid: 'https://orcid.org/0000-0002-2542-3582'
-repository-code: 'https://github.com/mehta-lab/VisCy'
-url: 'https://github.com/mehta-lab/VisCy'
+    orcid: "https://orcid.org/0000-0002-2542-3582"
+repository-code: "https://github.com/mehta-lab/VisCy"
+url: "https://github.com/mehta-lab/VisCy"
 abstract: computer vision models for single-cell phenotyping
 keywords:
   - machine-learning
```

Comment:
- image translation
- representation learning
- cell state segmentation

Reply:
Also these keywords are software identifiers more than research identifiers and were taken from existing GitHub topics. I propose only adding `image-translation` and `representation-learning` here.

Reply:
See https://github.com/mehta-lab/VisCy/pull/188/commits/bc6c4561329c669d854f24395e96df7760778a24. I also updated the GitHub topics.